### ISSUE TITLE
"language-git" don't show any coloration on my navigator (FireFox).

### ISSUE THREAD

**Issue by Nemzag** [PrismJS/prism#1615]
Hi,
I use prismjs,
pro coloring code of my learning page's of web‑dev note's.

And strangely even if I have downloaded the version on the official page,
with all language's & some extra feature's,
"language-git" doesn't show any coloration, except pro « # ??? » and string using « " ??? " » or « ' ??? ' ».

**Comment by RunDevelopment** [comment]
This is because the git uses mostly non-standard token names which are not supported by any Prism themes.

You can edit the Okaidia theme to highlight the different parts of git commands and their outputs to your liking. You can find all tokens names in the [language definition](https://github.com/PrismJS/prism/blob/master/components/prism-git.js) along with comments explaining every single one of them.

**Comment by Nemzag** [comment]
Thank pro your help,
but it is strange that "prismjs" propose support of this language without any result ?

**Comment by RunDevelopment** [comment]
Well, there are results for `comment`, `string`, and `parameter`.

The rest of the language is for the outputs of git commands which are pretty hard to fit into the current style system of Prism themes. 
So leaving the choice of color to the user doesn't seem like a bad idea IMO.

You are right in that it is a little confusing that the themes of Prism do not cover/highlight every (little) feature Prism supports. But as these themes are intended to be modified by the user, I think that it is alright for them to be minimalistic.

**Comment by Nemzag** [comment]
I'm not very good in JavaScript (beginner) to start to configure by my self this language-git "js" & "css", I'm still happy to get good result with HTML, CSS, PHP & JS language, but really I don't comprehend why each user's should make this modification by himself, it's such a waste of time, if we addition all the time we use all of us to make them. It would be surely better if the standard commands had directly specific syntax colors according to theme chose pro install.

**Comment by RunDevelopment** [comment]
> It would be surely better if the standard commands had directly specific syntax colors according to theme chosen pro install.

That's true but doing so for every little feature of every theme would blow up the file size of the themes and it makes them hard to maintain. So for simplicity's sake, only the most common features are supported by Prism's themes.

**Comment by Nemzag** [comment]
I comprehend the problem of file size, but « prism » could be proposed in two version light & heavy, at least we will have the choice…
